sdk/metrics/src/main/java/io/opentelemetry/sdk/metrics/internal/state/MetricStorageRegistry.java

```diff
@@ -38,7 +38,7 @@ public class MetricStorageRegistry {
   private final Map<MetricDescriptor, MetricStorage> registry = new HashMap<>();
 
   /** Returns a {@link Collection} of the registered {@link MetricStorage}. */
-  public Collection<MetricStorage> getMetrics() {
+  public Collection<MetricStorage> getStorages() {
```

Comment:
Rename for improved clarify.

Reply:
Makes sense. It could be even something like "getAllStorages" to make it clear that all registered ones will be returned.
It may also make sense to change JavaDocs - eg line 48 to "Registers the storage..." (I can't add review comment to these lines directly).